plugin-server/src/main/services/http-server.ts

```diff
@@ -126,8 +126,12 @@ export function createHttpServer(
 }
 
 function exportProfile(req: IncomingMessage, res: ServerResponse) {
-    // Mirrors golang's pprof behaviour of exposing on-off profiles through HTTP endpoints
-    // Port-forward a target pod and run: curl -vOJ "http://localhost:6738/_profile/cpu?seconds=30"
+    // Mirrors golang's pprof behaviour of exposing ad-hoc profiles through HTTP endpoints
+    // Port-forward pod 6738 on a target pod and run:
```

Comment:
for the n00bies give a cmd here too?

Reply:
This belong in a runbook instead of a probably-outdated code comment. For reference:

  - a specific pod: `kubectl port-forward pod/ingestion-plugins-webhooks-5bd55497b9-snpb4 6738:6738`
  - a random pod in a deployment `kubectl port-forward deployment/ingestion-plugins-onevent 6738:6738`